src/Sentry.Unity.Editor/Android/GradleSetup.cs

```diff
@@ -26,112 +23,44 @@ public GradleSetup(IDiagnosticLogger logger, string gradleProjectPath)
 
         public void UpdateGradleProject()
         {
-            _logger.LogInfo("Adding Sentry to the gradle project.");
-            var unityLibraryGradleContent = LoadGradleScript(_unityLibraryGradle);
-            unityLibraryGradleContent = InsertIntoScope(unityLibraryGradleContent, DependencyScopeName, SdkDependencies);
-            File.WriteAllText(_unityLibraryGradle, unityLibraryGradleContent);
+            _logger.LogDebug("Adding Sentry to the gradle project.");
```

Comment:
```suggestion
            _logger.LogInfo("Adding Sentry to the gradle project.");
```
I'd prefer this to stay `Info` to be visible for users by default.